esp_sysview/idf_component.yml

```diff
@@ -0,0 +1,7 @@
+version: 1.0.0
+description: SEGGER SystemView component for ESP-IDF
+url: https://github.com/espressif/idf-extra-components/tree/master/esp_sysview
+issues: https://github.com/espressif/idf-extra-components/issues
+repository: https://github.com/espressif/idf-extra-components.git
+dependencies:
+  idf: ">=6.1"
```

Comment:
we are backoprint this to 6.0 branch

Reply:
Yes. To pass the CI checks, I limited to 6.1 for a while.  Backport merge and sync can take time.